configuration/pih/htmlforms/hiv/section-pmtct-history.xml

```diff
@@ -133,6 +133,7 @@
                 setUpEdd('<lookup expression="encounter.getEncounterDatetime().getTime()"/>', '<uimessage code="pihcore.weeks"/>');
                 setUpNextButtonForSections('pmtct-history');
                 setUpExpandableContacts('<uimessage code="pihcore.invalid.phoneNumber"/>');
+                datepickerAsyncValidation('<uimessage code="pihcore.date.error.inTheFuture"/>','<uimessage code="pihcore.startDate.greather"/>');
```

Comment:
Small typo"  "greather" should be "greater"

Reply:
It's corrected